test/slots/Slot.test.js

```diff
@@ -86,7 +87,7 @@ describe('the Slot class', () => {
   describe('.assertStructure', () => {
 
     it('blows up because Slot is a virtual class and it wants to kindly tell you that a mistake was likely made in a child that has not defined the method', () => {
-      const slot = new Slot(InSlotSeeder.generate())
+      const slot = new Slot(FloatInSlotSeeder.generate())
```

Comment:
This may be sufficient and closer to what is being tested. `seedOne` is one of the extended methods from `FactorySeeder` which all the seeders have. It runs the seeder's `generate` followed by the seeders `SEED_METHOD` (which, in this case, is `new Slot`)

```suggestion
      const slot = SlotSeeder.seedOne()
```